package.json

```diff
@@ -33,7 +33,7 @@
     "body-parser": "1.20.3",
     "content-disposition": "0.5.4",
     "content-type": "~1.0.4",
-    "cookie": "0.6.0",
+    "cookie": "^0.7.1",
```

Comment:
LGTM

Please check this relevant comments as well. https://github.com/expressjs/express/pull/6017/files#r1789129519

My take: Please keep this change as is with the caret ^. It will make my life easier in the future.
Another reason is these express dependencies are probably the best I have seen adhering to the semver principles.

Thank you.

Reply:
Thanks for the suggestion @NewEraCracker, please check https://github.com/expressjs/express/pull/6029#discussion_r1791589983